docs/content/any/project/changelogs/NEXT.md

```diff
@@ -16,6 +16,12 @@ draft: true
 
 - Fixed a bug that caused the Go library to crash when using Go 1.17 on macOS.
 
+### Node.js
+
+#### Other bugs & improvements
+- The `Class` type for representing abstract resources for data filtering is
```

Comment:
This isn't really what the `Class` type is for, at least internally. Is this the only context in which someone would encounter the `Class` type while using the library?

Reply:
It's the only example on record afaik ...